src/api/helpers.jl

```diff
@@ -361,18 +361,25 @@ function h5p_get_efile_prefix(plist)
 end
 
 function h5p_get_external(plist, idx = 0)
-    name_size = Csize_t(1024)
-    name = zeros(UInt8, name_size)
-    offset = Ref{off_t}(0)
+    offset = Ref{Int64}(0)
```

Comment:
I think this should still be `off_t` following the C header:
https://portal.hdfgroup.org/display/HDF5/H5P_GET_EXTERNAL

Reply:
whops, thanks, I was locally testing and forgot to put it back in.